<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>驾驶行为分类</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="source_line"/>
     </item>
     <item>
      <widget class="QPushButton" name="source_btn">
       <property name="text">
        <string>选择源文件夹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dest_line"/>
     </item>
     <item>
      <widget class="QPushButton" name="dest_btn">
       <property name="text">
        <string>选择目标文件夹</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>163</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="last_btn">
       <property name="text">
        <string>上一图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="img_label">
       <property name="text">
        <string>图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next_btn">
       <property name="text">
        <string>下一图片</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>163</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="start_btn">
       <property name="text">
        <string>开始打标签</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="progress_label">
       <property name="text">
        <string>进度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="label_0">
       <property name="text">
        <string>正常</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="label_1">
       <property name="text">
        <string>侧视</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="label_2">
       <property name="text">
        <string>喝水</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="label_3">
       <property name="text">
        <string>吸烟</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="label_4">
       <property name="text">
        <string>操作中控</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="label_5">
       <property name="text">
        <string>玩手机</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="label_6">
       <property name="text">
        <string>侧身拿东西</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="label_7">
       <property name="text">
        <string>整理仪容</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="label_8">
       <property name="text">
        <string>接电话</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_btn">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
